app/controllers/openstax/utilities/status_controller.rb

```diff
@@ -6,7 +6,7 @@ class OpenStax::Utilities::StatusController < ActionController::Base
   layout 'openstax/utilities/status'
 
   def index
-    @application_name = Rails.application.class.parent_name
+    @application_name = OSU::SITE_NAME.humanize
     @environment_name = Rails.application.secrets.environment_name
```

Comment:
so the application class isn't `TutorApplication` or something like that any longer?

Reply:
It's still Tutor::Application but they renamed the parent_name method, it's now module_parent_name which is what OSU::SITE_NAME uses.